tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "allowSyntheticDefaultImports": true,
     "baseUrl": ".",
     "declaration": true,
-    "emitDeclarationOnly": true,
+    // "emitDeclarationOnly": true,
```

Comment:
intentional? or from running tsc or smth lol

Reply:
oh shoot, yep didn't mean to commit that. TY

Reply:
fixed in https://github.com/PrefectHQ/prefect-ui-library/pull/2607/commits/5d0f56cea9c1886a327784458eb20e49a0e13855